src/Analytics/Adapter/GoogleAnalytics.php

```diff
@@ -12,14 +12,14 @@
  * @license The MIT License (MIT) <http://www.opensource.org/licenses/mit-license.php>
  */
 
-namespace Utopia\Analytics;
+namespace Utopia\Analytics\Adapter;
 
 use Exception;
-use Utopia\Analytics\Analytics;
+use Utopia\Analytics\Adapter;
 
-class GoogleAnalytics extends Analytics
+class GoogleAnalytics extends Adapter
 {
-    private const URL = 'https://www.google-analytics.com/collect';
+    public $endpoint = 'https://www.google-analytics.com/collect';
```

Comment:
```suggestion
    private const $endpoint = 'https://www.google-analytics.com/collect';
```

Reply:
No, by making it private, we ensure no one from outside this class can access the member. 

For example if `$endpoint` were `public` I could do this


```php

$adapter = new Adapter();
$adapter->endpoint = "http://abc.com"
```

By making it private we ensure that the variable cannot be accessed directly.